src/components/checkboxCheckmark/CheckboxCheckmark.tsx

```diff
@@ -0,0 +1,162 @@
+import * as React from "react";
+import styled, { css } from "styled-components";
+import theme from "../../theme";
+import { Flex } from "../container";
+import {
+  ExtendedTooltip,
+  ExtendedTooltipProps,
+} from "../extendedTooltip/ExtendedTooltip";
+import { SVGIcon } from "../svgicon";
+
+const SingleCheckWrapper = styled(Flex)`
+  margin-bottom: ${theme.spacing.spacing04};
```

Comment:
remove margins from this wrapper, lets have more control over this outside the component